cli_tools/mocks/mock_storage_client.go

```diff
@@ -1,17 +1,3 @@
-//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
generating mocks removes copyright notice by default so you have to add it manually back. Add it also for the new mocks you generated.